engines/sludge/keymapper_tables.h

```diff
@@ -0,0 +1,556 @@
+
+        act = new Action("SHOWBOXES", _("DEBUG: Show boxes"));
+        act->setKeyEvent(KeyState(KEYCODE_b, 'b'));
+        act->addDefaultInputMapping("b");
+        act->addDefaultInputMapping("JOY_LEFT");
+        debugKeyMap->addAction(act);
+
+    } else if (gameId == "atw") {
+        act = new Action(kStandardActionLeftClick, _("Move / Interact"));
+        act->setLeftClickEvent();
+        act->addDefaultInputMapping("MOUSE_LEFT");
+        act->addDefaultInputMapping("JOY_A");
+        gameKeyMap->addAction(act);
+
+        act = new Action(kStandardActionRightClick, _("Inventory"));
+        act->setRightClickEvent();
+        act->addDefaultInputMapping("MOUSE_RIGHT");
+        act->addDefaultInputMapping("JOY_B");
+        gameKeyMap->addAction(act);
+
+        act = new Action("QUIT", _("Quit"));
+        act->setKeyEvent(KeyState(KEYCODE_ESCAPE, ASCII_ESCAPE));
+        act->addDefaultInputMapping("ESCAPE");
+        act->addDefaultInputMapping("q");
+        act->addDefaultInputMapping("JOY_BACK");
+        gameKeyMap->addAction(act);
+    
+    } else if (gameId == "cubert") {
+        act = new Action(kStandardActionLeftClick, _("Move / Interact"));
+        act->setLeftClickEvent();
+        act->addDefaultInputMapping("MOUSE_LEFT");
+        act->addDefaultInputMapping("JOY_A");
+        gameKeyMap->addAction(act);
+
+        act = new Action(kStandardActionRightClick, _("Inventory / Examine"));
+        act->setRightClickEvent();
+        act->addDefaultInputMapping("MOUSE_RIGHT");
+        act->addDefaultInputMapping("JOY_B");
+        gameKeyMap->addAction(act);
+
+        act = new Action(kStandardActionPause, _("Pause"));
+        act->setKeyEvent(KeyState(KEYCODE_ESCAPE, ASCII_ESCAPE));
+        act->addDefaultInputMapping("ESCAPE");
+        act->addDefaultInputMapping("JOY_START");
+        gameKeyMap->addAction(act);
+
+    } else if (gameId == "frasse") {
+        act = new Action(kStandardActionLeftClick, _("Move / Interact"));
+        act->setLeftClickEvent();
+        act->addDefaultInputMapping("MOUSE_LEFT");
+        act->addDefaultInputMapping("JOY_A");
+        gameKeyMap->addAction(act);
+
+        act = new Action(kStandardActionRightClick, _("Inventory"));
+        act->setRightClickEvent();
+        act->addDefaultInputMapping("MOUSE_RIGHT");
+        act->addDefaultInputMapping("JOY_B");
+        gameKeyMap->addAction(act);
+
+        act = new Action(kStandardActionPause, _("Pause"));
+        act->setKeyEvent(KeyState(KEYCODE_ESCAPE, ASCII_ESCAPE));
+        act->addDefaultInputMapping("ESCAPE");
+        act->addDefaultInputMapping("JOY_START");
+        gameKeyMap->addAction(act);
+
+        act = new Action("SKIPDLG", _("Skip dialogue"));
+        act->setKeyEvent(KeyState(KEYCODE_PERIOD, '.'));
+        act->addDefaultInputMapping("PERIOD");
+        act->addDefaultInputMapping("JOY_X");
+        gameKeyMap->addAction(act);
+
+    } else if (gameId == "tgttpoacs") {
```

Comment:
There's a minigame located in the Arcade called "Wizard Warble" that uses some additional keys that aren't currently listed.